pages/tutorials.js

```diff
@@ -48,7 +48,7 @@ export async function getStaticProps() {
   // get slug and frontmatter
   const tutorials = files.map(filename => {
     // create the slug
-    const slug = filename.replace('.md', '')
+    const slug = filename.replace('.mdx', '')
```

Comment:
My idea here was to write a function which extracts the extensions, something like this:

```javascript
export function getExtension(filename) {
       return filename.split('.').pop();
```

And then to use it inside the `getStatic ...` functions.

Do you think this can be a strategy?
I think that my code above has some errors that I need to find.

Reply:
Do not think about this yet, I'm trying to understand how to write something useful using `js` and the two `GetStatic` functions. I let you know when I will have a solution.

Reply:
However, this problem is not strictly related to this PR, my bad.

Please, open a dedicated issue, and resolve this and related conversations. Let's merge as soon as possible, later on we will fix also this :)

Reply:
oke. What is missing, now, before the merging?